src/open_samus_returns_rando/cli.py

```diff
@@ -60,9 +61,12 @@ def main() -> None:
     with args.input_json.open() as f:
         configuration = json.load(f)
 
+    start = time.time()
     samus_returns_patcher.patch_extracted(
         args.input_path,
         args.input_exheader,
         args.output_path,
         configuration,
     )
+    end = time.time()
+    print(f"Patcher took {end - start:.03f} seconds")
```

Comment:
Methinks this should not be in release builds

Reply:
It exists for odr so 🤷‍♂️

Reply:
Randovania doesn't use this function and it's convenient when running manually.

Reply:
It still shows up in randovania builds via the cmd console that pops up on windows tho, no?